syntax = "proto2";

message DevicesListRequest {
}
